src/shared/utils/athenaV3.ts

```diff
@@ -7,17 +7,21 @@ import {
 } from "@aws-sdk/client-athena";
 
 const INTERVAL_MS = 1000;
+const MAX_ATTEMPTS = 10;
 
 export class AthenaQueryExecutor {
   athena: AthenaClient;
-
   intervalMillis: number;
+  maxAttempts: number;
 
-  MAX_ATTEMPTS = 10;
-
-  constructor(athena: AthenaClient, intervalMillis = INTERVAL_MS) {
+  constructor(
+    athena: AthenaClient,
+    intervalMillis = INTERVAL_MS,
+    maxAttempts = MAX_ATTEMPTS
```

Comment:
You've made maxAttempts configurable, but the only caller to this constructor doesn't set it...  Just confirming that you didn't mean to change that as well.

Reply:
Yeah it just seemed like the kinda thing there should be an option for